src/NodeAutoCompleteViewExtension/Properties/Resources.Designer.cs

```diff
@@ -59,5 +59,41 @@ internal Resources() {
                 resourceCulture = value;
             }
         }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Cluster.
+        /// </summary>
+        public static string Cluster {
+            get {
+                return ResourceManager.GetString("Cluster", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Node AutoComplete View Extension.
+        /// </summary>
+        public static string ExtensionName {
+            get {
+                return ResourceManager.GetString("ExtensionName", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Node Type Match.
+        /// </summary>
+        public static string NodeTypeMatch {
```

Comment:
Is our control auto-resizable horizontally ? This text will probably be bigger in certain languages.

Reply:
The whole flyout has a fixed size but the NodeTypeMatch textview is sized as `Auto`, so it will push the other views sized as `*` (the buttons) to make space for itself while possible)